### PR TITLE
update news vbump to 0.8.5, [skip vbump]

### PR DESCRIPTION
**Pull request by shajoezhu**
# Pull Request

Fixes #nnn

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
-# tern 0.8.4.9005
+# tern 0.8.5
 
-### Enhancements
-* Added method for `character` class to `h_coxreg_inter_effect` enabling `character` covariates in `summarize_coxreg`.
+### Miscellaneous
```
Comment (edelarua) [inline NEWS.md:3]:
```suggestion
### Enhancements
```

File: NEWS.md [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
 * Started deprecation cycle for `summarize_vars` and `control_summarize_vars`. Renamed into `analyze_vars` and `control_analyze_vars` to reflect underlying `rtables` machinery while keeping backward compatibility with aliases.
 * Added `ylim` argument to `g_km` to allow the user to set custom limits for the y-axis.
 * Added assertion to `g_km` which checks whether there is one arm present in the data when `annot_coxph` is true.
 * Added `flag_labels` argument to `s_count_patients_with_flags` to enable more label handling options in `count_patients_by_flags`.
```
Comment (edelarua) [inline NEWS.md:7]:
```suggestion
* Added `ylim` argument to `g_km` to allow the user to set custom limits for the y-axis.
* Added assertion to `g_km` which checks whether there is one arm present in the data when `annot_coxph` is true.
* Added `flag_labels` argument to `s_count_patients_with_flags` to enable more label handling options in `count_patients_by_flags`. 

### Miscellaneous
* Started deprecation cycle for `summarize_vars` and `control_summarize_vars`. Renamed into `analyze_vars` and `control_analyze_vars` to reflect underlying `rtables` machinery while keeping backward compatibility with aliases.
```

**Review by edelarua — CHANGES_REQUESTED** [review]
Just a suggestion to improve organization of the NEWS file

**Review by edelarua — APPROVED** [review]
Approving to unblock

**Comment by shajoezhu** [comment]
Thanks a lot @edelarua

**Pull request merged**